roles/network_plugin/calico/tasks/main.yml

```diff
@@ -41,7 +41,7 @@
   notify: restart calico-node
 
 - name: Calico | Copy cni plugins from hyperkube
-  command: "{{ docker_bin_dir }}/docker run --rm -v /opt/cni/bin:/cnibindir {{ hyperkube_image_repo }}:{{ hyperkube_image_tag }} /usr/bin/rsync -a /opt/cni/bin/ /cnibindir/"
+  command: "{{ docker_bin_dir }}/docker run --rm -v /opt/cni/bin:/cnibindir {{ hyperkube_image_repo }}:{{ hyperkube_image_tag }} /usr/bin/rsync -ac /opt/cni/bin/ /cnibindir/"
```

Comment:
It does checksums of ~12 files.
If the destination doesn't exist, the comparison is: 1.81s for rsync -a, 1.99s for rsync -ac
If the destination does exist, the comparison is: 0.66s for rsync -a, 1.44s for rsync -ac.

We couldn't hope to beat that with Ansible stat module, but with -ac, it doesn't overwrite.